CS3219-AY2223S1/cs3219-project-ay2223s1-g35
# Fr 1.6

Things done:

- [X] Add linkage between sign up and login
- [X] Refactor code base to standardize some formatting
- [X] Dummy profile page to view info of user in the future
- [x] Change password backend + DB changes

To test:
Open up dev console, login with a normal user.
When changing password, check console for feedback on whether it works.
Re-login with new password.
